tutorcairn/templates/cairn/apps/openedx/scripts/server.py

```diff
@@ -0,0 +1,47 @@
+from aiohttp import web
+import subprocess
+import logging
+from opaque_keys.edx.locator import CourseLocator
+
+# Configure logging
+logging.basicConfig(level=logging.INFO,
+                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# Get a logger instance
```

Comment:
no need for the logging comments here, the code is self-explanatory.

Reply:
Updated.